deploy/olm-catalog/openshift-pipelines-operator/0.9.1/openshift-pipelines-operator.v0.9.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,388 @@
+    capabilities: Basic Install
+    categories: Developer Tools, Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/openshift-pipeline/openshift-pipelines-operator:v0.9.1
+    createdAt: "2019-03-15T19:44:21Z"
+    description: OpenShift Pipelines is a cloud-native CI/CD solution for building
+      pipelines using Tekton concepts which run natively on OpenShift and Kubernetes.
+    operators.operatorframework.io/internal-objects: '["config.operator.tekton.dev"]'
+    repository: https://github.com/openshift/tektoncd-pipeline-operator
+    support: Red Hat, Inc.
+  name: openshift-pipelines-operator.v0.9.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: OpenShift Pipelines is a cloud-native CI/CD solution for building
+        pipelines using Tekton concepts which run natively on OpenShift and Kubernetes.
+      displayName: OpenShift Pipelines Config
+      kind: Config
+      name: config.operator.tekton.dev
+      resources:
+      - kind: clusterroles
+        name: ""
+        version: v1
+      - kind: customresourcedefinitions
+        name: ""
+        version: v1beta1
+      - kind: deployments
+        name: ""
+        version: v1beta1
+      - kind: podsecuritypolicies
+        name: ""
+        version: v1beta1
+      - kind: clusterrolebindings
+        name: ""
+        version: v1beta1
+      - kind: services
+        name: ""
+        version: v1
+      - kind: pods
+        name: ""
+        version: v1
+      - kind: configmaps
+        name: ""
+        version: v1
+      - kind: config
+        name: ""
+        version: v1alpha1
+      - kind: namespaces
+        name: ""
+        version: v1
+      - kind: serviceaccounts
+        name: ""
+        version: v1
+      specDescriptors:
+      - description: Namespace where OpenShift Pipelines is installed
+        displayName: Target Namespace
+        path: targetNamespace
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      statusDescriptors:
+      - description: Status of OpenShift Pipelines Installation
+        displayName: Installation Status
+        path: conditions[0].code
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Version of OpenShift Pipelines installed
+        displayName: OpenShift Pipelines Version
+        path: conditions[0].version
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      version: v1alpha1
+  description: |
+    OpenShift Pipelines is a cloud-native continuous integration and delivery
+    (CI/CD) solution for building pipelines using [Tekton](https://tekton.dev).
+    Tekton is a flexible Kubernetes-native open-source CI/CD framework which
+    enables automating deployments across multiple platforms (Kubernetes,
+    serverless, VMs, etc) by abstracting away the underlying details.
+
+    ## Features
```

Comment:
May like add note about new cluster tasks?

Reply:
This version of Operator (CSV) is already published in OperatorHub. We can incorporate this suggestion in the next release.

I have captured this suggestion in an issue.
https://github.com/openshift/tektoncd-pipeline-operator/issues/175

Reply:
@hrishin